### PR TITLE
update rest spec

### PR DESCRIPTION
**Pull request by gangtao**
I am updating the spec and generated code to the latest neutron 1.4.4.

### CODE REVIEW

File: python/swagger_client/models/alert.py [hunk 1]
```diff
@@ -373,8 +373,8 @@ def resolve_query(self, resolve_query):
         :param resolve_query: The resolve_query of this Alert.  # noqa: E501
         :type: Query
         """
-        # if resolve_query is None:
-        #     raise ValueError("Invalid value for `resolve_query`, must not be `None`")  # noqa: E501
+        if resolve_query is None:
```
Comment (ye11ow) [inline python/swagger_client/models/alert.py]: I believe it was manually commented out before.
We should track those changes and make sure we apply them every time

File: python/swagger_client/models/ingest_data.py [hunk 2]
```diff
@@ -42,10 +42,8 @@ def __init__(self, columns=None, data=None):  # noqa: E501
         self._columns = None
         self._data = None
         self.discriminator = None
-        if columns is not None:
```
Comment (ye11ow) [inline python/swagger_client/models/ingest_data.py:45]: was it modified by us?

Comment (gangtao) [reply python/swagger_client/models/ingest_data.py:45]: No, this is generated code

File: python/swagger_client/models/alert.py (hunk 1 above)
Comment (gangtao) [reply python/swagger_client/models/alert.py]: yes, I remember I blocked that due to test failure

**Review by ye11ow — APPROVED** [review]

**Comment by jovezhong** [comment]
(Jove Github Bot) added it to the current sprint.

**Comment by jovezhong** [comment]
(Jove Github Bot) moved this ticket out of the GitHub project(up to 1200 tickets for one project).

**Pull request merged**